### ISSUE TITLE
Test setting up new instance on a fresh vm

### ISSUE THREAD

**Issue by physikerwelt** [DRMF/DRMF#73]
@plato2000 I know that it might sould mean to you... but I think it would be really good to recreate the drmf2017 instance and install everything again exactly following the documentation.
That way we can be sure that we have a setup that works.
@HowardCohl what is your oppinion on that?

**Comment by HowardCohl** [comment]
@plato2000 @physikerwelt Yes, I completely agree.

**Comment by physikerwelt** [comment]
@plato2000 please give me green light, if I can recreate the instance. PS: Do you have a device that you could use for two factor auth (then I can assign the rights to you that you can cerate the instance by yourself)

**Comment by plato2000** [comment]
@physikerwelt I agree, at this point I don't know what's wrong and this is probably the best course of action. I do have 2-step verification set up in the wikitech settings.

**Comment by physikerwelt** [comment]
ok, you are project admin know. Please ask if you are unsure about something?
